modules/search_api_solr_autocomplete/tests/src/Kernel/SearchApiSolrAutocompleteTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\Tests\search_api_solr_autocomplete\Kernel;
+
+use Drupal\search_api\Entity\Server;
+use Drupal\search_api_autocomplete\Entity\Search;
+use Drupal\Tests\search_api_solr\Kernel\SearchApiSolrTest;
+
+/**
+ * Tests search autocomplete results using the Solr search backend.
+ *
+ * @group search_api_solr
+ */
+class SearchApiSolrAutocompleteTest extends SearchApiSolrTest {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'search_api_solr_autocomplete',
+  ];
```

Comment:
I assume search_api_autocomplete is missing here.

Reply:
@mkalkbrenner  Yes, looks like. Also, I'm wondering why `$modules` is public. It should be `protected`